opentech/apply/users/tests/test_oauth_access.py

```diff
@@ -28,12 +28,12 @@ def test_oauth_user_email_not_whitelisted(self):
         response = self.client.get(reverse('users:oauth'), follow=True)
         self.assertEqual(response.status_code, 403)
 
-    @override_settings(SOCIAL_AUTH_GOOGLE_OAUTH2_WHITELISTED_DOMAINS=['email.com'])
-    def test_oauth_whitelisted_user_can_see_link_to_oauth_settings_page(self):
-        self.login()
+    # @override_settings(SOCIAL_AUTH_GOOGLE_OAUTH2_WHITELISTED_DOMAINS=['email.com'])
+    # def test_oauth_whitelisted_user_can_see_link_to_oauth_settings_page(self):
```

Comment:
Is it better to run this test and assert not contains. Otherwise we wont necessarily remember to re-enable this test.

`self.assertNotContains(response, 'Manage OAuth')`